core/src/main/scala/com/typesafe/tools/mima/core/PicklePrinter.scala

```diff
@@ -141,13 +141,13 @@ object PicklePrinter {
   /** Returns the buffer as a sequence of (Int, Array[Byte]) representing
    *  (tag, data) of the individual entries.  Saves and restores buffer state.
    */
-  private def toIndexedSeq(buf: PickleBuffer): IndexedSeq[(Int, Array[Byte])] = {
+  private def toIndexedSeq(buf: PickleBuffer): IndexedSeq[(Int, Array[Byte])] = (
     for (idx <- buf.createIndex) yield {
       buf.atIndex(idx) {
         val tag = buf.readByte()
         val len = buf.readNat()
         tag -> buf.bytes.slice(buf.readIndex, buf.readIndex + len)
       }
     }
-  }
+  ).toIndexedSeq
```

Comment:
What's going on here?  Looks like there's an implicit conversion from Array to immutable IndexedSeq??  Wow.

Reply:
It is obvious in some contexts but not this one. There is not a good "just wrap my array" solution that cross-compiles.

I was about to tweak the loop to iterate once over buf to build an IndexedSeq but maybe that is overkill, and more of just an exercise one does at midnight when time seems momentarily infinite.